config/environments/production.rb

```diff
@@ -43,7 +43,12 @@
   # `config.assets.precompile` and `config.assets.version` have moved to config/initializers/assets.rb
 
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  # config.action_controller.asset_host = "http://assets.example.com"
+  if ENV["PUBLIC_URL"].present?
```

Comment:
`PUBLIC_URL` is an environment variable that is consumed by Create-React-App to know where the primary assets are being hosted from. 

We updated the [Fastly configuration in this pull request](https://github.com/brave/devops/pull/902).